src/components/Tab/style.scss

```diff
@@ -6,19 +6,18 @@
   flex-direction: row;
   align-items: center;
   padding: 0 24px;
-  height: 49px;
```

Comment:
I wonder why it was static. I hope it was not relative with any IE/EDGE issue ;x

Reply:
Checked and looks fine after discussion with @kamilmateusiak